_data/cryptocurrencies.yml

```diff
@@ -323,6 +323,15 @@ websites:
       - totp
     doc: https://support.coins.ph/hc/en-us/articles/202604924
 
+  - name: CoinSmart
+    url: https://www.coinsmart.com/
+    img: coinsmart.svg
+    tfa:
+      - totp
+    doc: https://coinspot.zendesk.com/hc/en-us/sections/115000049254
+    regions:
```

Comment:
I don't think a regions array is required here - the site seems to allow users from different countries to register on the sign up page and the terms of service mention that users outside of Canada can sign up.